### ISSUE TITLE
Add video map toggle option

### ISSUE THREAD

**Issue by headphase** [openscope/openscope#444]
I can appreciate the realism that video maps provide to experienced players and actual controllers, but from a layman's perspective, they are really mystifying and detract from/clutter the experience.  It would be nice to have an option of hiding them like the other overlay components.

**Comment by ij1** [comment]
As a layman myself, I certainly agree with that sentiment. Seeing some of the airports that currently come with video maps for the first time made me feel very confused. In addition, back then it was with my old computer with not that new browser too so UI was also noticeably slower than airports w/o video maps.

Only after finding some reference to "video maps" in some github issue I understood what that mess actually "is". That is, not that I still know all details drawn there but now after trying to create an airport json myself I can at least figure out what some (most) of the components seem to be.

I've few additional points:
- "Video map" is not a term known to a layman so UI toggle using it won't be understood. Perhaps tutorial could be extented to introduce it (that one can toggle it off/on and mention that it's what appears in the realistic working environment).
- In addition, some simple kind of explanation in docs on what is being displayed by the video map would be useful so that any layman can understand the purpose of the clutter so there would then perhaps be less incentive to turn it off. But of course it would require that somebody comes up with that.

**Comment by n8rzz** [comment]
I agree on all counts.  I've added #446 for the documentation bit.
